source/documentation/device_wifi.md

```diff
@@ -0,0 +1,28 @@
+# Certificate Authentication (Alpha)
+
+## What is Certificate Authentication
+Device-Wifi requires a user to have either a government email address, be sponsored by someone with a government email address, or have a mobile phone number. Device-Wifi only requires that a certificate needs to be installed on a device to gain access, this removes the need to always enter an email and password to gain access to government Wifi.
```

Comment:
Does it really require that they have government email addresses? Where is this enforced?

Reply:
Im sure we only want users with government emails using GovWifi anyway?